source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -624,24 +624,23 @@ message TransformationAddNoContractionDecoration {
 
 }
 
-message TransformationAddParameters {
+message TransformationAddParameter {
 
-  // Adds new parameters into the function.
+  // Adds a new parameter into the function.
 
   // Result id of the function to add parameters to.
   uint32 function_id = 1;
 
-  // New type of the function.
-  uint32 new_type_id = 2;
+  // Fresh id for a new parameter.
+  uint32 parameter_fresh_id = 2;
 
-  // Type ids of parameters to add to the function.
-  repeated uint32 new_parameter_type = 3;
+  // Result id of the instruction, used to initializer new parameter
+  // in function calls. Type id of that instruction is the type id of the parameter.
+  // It may not be OpTypeVoid.
+  uint32 initializer_id = 3;
 
-  // Result ids for new parameters.
-  repeated uint32 new_parameter_id = 4;
-
-  // Constants to initialize new parameters from.
-  repeated uint32 constant_id = 5;
+  // A fresh id for a new function type. This might not be used.
```

Comment:
Elaborate briefly on why it might not be used (due to the new function type already existing).

Reply:
Fixed in 93b8e3a.